packages/office-ui-fabric-react/src/components/Panel/Panel.styles.ts

```diff
@@ -153,8 +153,8 @@ const getPanelBreakpoints = (type: PanelType): { [x: string]: IStyle } | undefin
 const commandBarHeight = '44px';
 
 const sharedPaddingStyles = {
```

Comment:
Is there where the breakpoints you're referring to in the descriptions used to be? I'm a bit confused on what you mean by "breakpoint determined style changes"

Reply:
replied in Teams